### PR TITLE
Reformatted and rearranged the unformatted and disarranged xml layout files

### LINKED ISSUE #107: Fix the disarranged and unformatted xml layout files
**Actual Behaviour**
Although making PR's forces this invariant that the code is rearranged and reformatted in neurolab-android still we currently have activity_about__us.xml, fragment_feedback_settings.xml, nav_header_main.xml files with unformatted and disarranged code.

**Expected Behaviour**
The code should always be properly arranged and formatted as it helps in debugging and readability for the developers.

**Would you like to work on the issue?**
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by suresh-gandhi**
Fixes #107 

**Changes**: 
- Reformatted and rearranged the files activity_about__us.xml, fragment_feedback_settings.xml, nav_header_main.xml.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by jddeep** [comment]
Is this PR even necessary in the current development process? We should focus more on implentation of the app features. Changes like this can be made once the App reaches a minimum stable state.

File: app/src/main/res/layout/activity_about__us.xml [hunk 1]
```diff
@@ -3,8 +3,8 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:background="@drawable/hill_background"
     android:layout_height="match_parent"
+    android:background="@drawable/hill_background"
```
Comment (jddeep) [inline app/src/main/res/layout/activity_about__us.xml:7]: Is this change really required? I am no expert in VCS but won't these cause mere Merge conflicts.

Comment (suresh-gandhi) [reply app/src/main/res/layout/activity_about__us.xml:7]: I know you are not an expert in VCS. No, they won't cause merge conflicts. And if they do we can also resolve them.

**Comment by suresh-gandhi** [comment]
> Is this PR even necessary in the current development process? 

I would suggest you see the PR template and carefully understand the 3rd point in the checklist.

>We should focus more on implementation of the app features.

Refer to the #79. I think we both should work on subsequent program modes for concrete implementation. If you are still unable to run the neurolab-desktop application message me privately on gitter. I would be willingly helping you. Lets do the work of contributing together and leave the merging and review to mentors :)
Cheers.

**Comment by jddeep** [comment]
@suresh-gandhi the neurolab-desktop.works fine on my machine. Thanks. I didn't talk about whether it should be merged or unmerged. We should definitely keep that decision to the mentors. I gave the merge-conflict advice to you from my past experience. And it not only about us. It is about other developers as well. Please don't take suggestions otherwise. Hope you understand buddy.

**Comment by jddeep** [comment]
@suresh-gandhi Ping me on gitter as I can see you need help in making quality contributions. Cheers mate!

**Comment by suresh-gandhi** [comment]
> @suresh-gandhi the neurolab-desktop.works fine on my machine. Thanks. I didn't talk about whether it should be merged or unmerged. We should definitely keep that decision to the mentors. I gave the merge-conflict advice to you from my past experience. And it not only about us. It is about other developers as well. Please don't take suggestions otherwise. Hope you understand buddy.

I understand brother. No, I am not taking it otherwise. I thought you were. Anyways, let's gear together and start working towards the repo buddy ;)

**Comment by jddeep** [comment]
@suresh-gandhi Great decision brother. let's not extend our discussion here. Glad you decided to contribute in quality towards the repo. Take inspiration and help from past PRs from other developers and start your quality work. :) And if you need any sort of help or want to discuss something feel free to ping me on gitter. Cheers buddy!

File: app/src/main/res/layout/activity_about__us.xml (hunk 1 above)
Comment (jddeep) [reply app/src/main/res/layout/activity_about__us.xml:7]: Let's not be too sure about that buddy. ;) You will understand.

**Review by mariobehling — APPROVED** [review]
Thanks for that, but let's focus on functionalities from here on. There will always be be improvements to code layout but it does not make the app more functional.
